index.html

```diff
@@ -521,52 +589,69 @@ <h2>Team</h2>
 
           <div class="col-lg-6">
             <div class="member d-flex align-items-start" data-aos="zoom-in" data-aos-delay="100">
-              <div class="pic"><img src="assets/img/team/Shubham.jpg" class="img-fluid" alt=""></div>
+              <div class="pic"><img src="assets/img/team/Aditya_Piyush.jpg" class="img-fluid" alt=""></div>
               <div class="member-info">
-                <h4>Shubham Kakde</h4>
+                <h4>Aditya Piyush</h4>
                 <span>Governor</span>
-                <p>Final Year B.Tech, Industrial and Systems Engineering (2017-2022)</p>
+                <p>Dept of Computer Science and Engineering (2019-2023)</p>
```

Comment:
When using the short form of 'department', it should be 'Dept.' (with a period in the end), not 'Dept'.